x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -33261,6 +33261,7 @@
     "xpack.remoteClusters.form.errors.illegalCharacters": "从名称中删除{characterListLength, plural, other {字符}} {characterList}。",
     "xpack.remoteClusters.form.errors.illegalSpace": "名称中不允许使用空格。",
     "xpack.remoteClusters.form.errors.nameMissing": "“名称”必填。",
+    "xpack.remoteClusters.form.errors.maxValue": "此数字必须等于或小于 {maxValue}。",
```

Comment:
How do we have the translations already?

Reply:
i reused the ones from Ingest_pipelines (`"xpack.ingestPipelines.pipelineEditor.communityId.seedMaxNumberError"`)